### PR TITLE
.NET Core 3 Preview 1 reqs

### LINKED ISSUE #9391: Update prereqs for Core 3.0 Preview 1
MacOS, Linux, and Windows prereqs topics need to be updated for 3.0

- [x] Windows: https://docs.microsoft.com/en-us/dotnet/core/windows-prerequisites
- [x] macOS: https://docs.microsoft.com/en-us/dotnet/core/macos-prerequisites
- [x] Linux: https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites

### PR DESCRIPTION
**Pull request by Thraka**
## Summary

Update the prerequisites information about Core 3.0 preview 1

Fixes #9391

### CODE REVIEW

**Comment by Thraka** [comment]
@mairaw do we have a tab name/id for Core 3.0 Preview 1?

File: docs/core/linux-prerequisites.md [hunk 1]
```diff
@@ -17,6 +17,33 @@ This article shows the dependencies needed to develop .NET Core applications on
 
 ## Supported Linux versions
 
+# [.NET Core 3.0 Preview 1](#tab/netcore30p1)
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: I believe the name I submitted for the tab name is netcore30

Also, it concerns me a bit that the Preview 1 would be the default tab here with this change. Shouldn't we show 2.0 first?

Comment (Thraka) [reply docs/core/linux-prerequisites.md]: Yes... I'll put it AFTER 1.x since it's a preview?

**Comment by mairaw** [comment]
The whitelist is here: https://docs.microsoft.com/en-us/contribute/validation-reference/tabbed-conceptual#tab-whitelist 

You can ask Megan for a new one, if you want one specific for Preview. I don't know if the validation tool validates both tab name and tab id or just tab id. You could try using netcore30 with a Preview 1 name and see if that would fail.

File: docs/core/linux-prerequisites.md (hunk 1 above)
Comment (mairaw) [reply docs/core/linux-prerequisites.md]: I think that's ok. It's kind of unfortunate we can't set each one is the default and have a proper order.

**Comment by mairaw** [comment]
I haven't actually reviewed this one as a whole. We usually wait until we get an approved PR unless we're pressed for time.

**Comment by Thraka** [comment]
Oh sorry, my brain thought you had approved it and we were just waiting on the tab id name. If you want to review it (it's not big) I can get any fixes in right away.

**Pull request merged**